api/api/controllers/test/test_active_response_controller.py

```diff
@@ -0,0 +1,50 @@
+import sys
+from unittest.mock import ANY, AsyncMock, MagicMock, call, patch
+
+import pytest
+from aiohttp import web_response
+
+with patch('wazuh.common.wazuh_uid'):
+    with patch('wazuh.common.wazuh_gid'):
+        sys.modules['wazuh.rbac.orm'] = MagicMock()
+        import wazuh.rbac.decorators
+        from api.controllers.active_response_controller import run_command
+        from wazuh import active_response
+        from wazuh.tests.util import RBAC_bypasser
+        wazuh.rbac.decorators.expose_resources = RBAC_bypasser
+        del sys.modules['wazuh.rbac.orm']
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('mock_request', [{'token_info': {'rbac_policies': 'rbac_policies_value'}}])
+async def test_active_response_controller(mock_request):
+    """Test all active_response_controller endpoints"""
+    async def test_run_command():
+        calls = [call(f=active_response.run_command,
+                      f_kwargs=ANY,
+                      request_type='distributed_master',
+                      is_async=False,
+                      wait_for_complete=False,
+                      logger=ANY,
+                      broadcasting=True,
+                      rbac_permissions=mock_request['token_info']['rbac_policies']
+                      )
+                 ]
+        result = await run_command(request=mock_request)
+        mock_dapi.assert_has_calls(calls)
+        mock_exc.assert_called_once_with(mock_dfunc.return_value)
+        assert isinstance(result, web_response.Response)
+
+    # Function list containing all sub tests declared above.
+    functions = [test_run_command()
+                 ]
+    for test_funct in functions:
+        with patch('api.controllers.active_response_controller.DistributedAPI.__init__',
+                   return_value=None) as mock_dapi:
```

Comment:
Why don't you mock the DAPI module using a decorator? It would apply to all tests and you reduce indentation.